docs/content/users/install/ddev-installation.md

```diff
@@ -165,15 +175,21 @@ Installing and upgrading DDEV are nearly the same thing, because you're upgradin
         
         If you prefer to use another Ubuntu distro, just install it and set it as default. For example, `wsl --set-default Ubuntu-22.04`.
 
-    2. Install Docker Desktop. If you already have chocolatey, `choco install -y docker-desktop` or [download Docker Desktop from Docker](https://www.docker.com/products/docker-desktop/).
-    3. Start Docker Desktop. You should now be able to do `docker ps` in PowerShell or Git Bash.
-    4. In `Docker Desktop -> Settings -> Resources -> WSL2 Integration` verify that Docker Desktop is integrated with your distro.
-    5. In an administrative `PowerShell` (5) run [this PowerShell script](https://raw.githubusercontent.com/drud/ddev/master/scripts/install_ddev_wsl2_docker_desktop.ps1) by executing:
+    2. Visit the Microsoft Store and install the *updated* "Windows Subsystem for Linux", then click "Open". It will likely prompt you for a username and password for the Ubuntu WSL2 instance it creates.
+
+    3. In "Windows Update Settings → Advanced Options" enable "Receive updates for other Microsoft products", and you may want to occasionally `wsl.exe --update` as well.
+
+    4. Install Docker Desktop. If you already have chocolatey, `choco install -y docker-desktop` or [download Docker Desktop from Docker](https://www.docker.com/products/docker-desktop/).
+    5. Start Docker Desktop. You should now be able to do `docker ps` in PowerShell or Git Bash.
+    6. In `Docker Desktop -> Settings -> Resources -> WSL2 Integration` verify that Docker Desktop is integrated with your distro.
+    7. In an administrative `PowerShell` (5) run [this PowerShell script](https://raw.githubusercontent.com/drud/ddev/master/scripts/install_ddev_wsl2_docker_desktop.ps1) by executing:
```

Comment:
Remove backticks from “PowerShell” so it’s consistent with the previous step, and remove `(5)` whose purpose is unclear. If it referenced step 5, it didn’t offer any perceived benefit—if anything we’d probably want to explain to someone how they’d even run an administrative PowerShell but I take it we assume the reader is comfortable with this.

```suggestion
    7. In an administrative PowerShell run [this PowerShell script](https://raw.githubusercontent.com/drud/ddev/master/scripts/install_ddev_wsl2_docker_desktop.ps1) by executing:
```

Reply:
Unfortunately, the new PowerShell promoted by MS as an aftermarket install does not work with this script, so the need is for PowerShell v5 (the built-in PowerShell).

Reply:
Ohhh, okay. Then maybe `(5)` could be `(v5!)` to clarify it’s describing version number and that the docs are pointing it out with some urgency?